### PR TITLE
[SEGA NN] NodeNameList chunk spec

### PR DESCRIPTION
**Pull request by TheExentist151**

### CODE REVIEW

File: hedgedocs/docs/nn/common/chunk-format/nodenamelist.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    char* Name;
+};
+```
+
+## NNE_NODENAME_SORTTYPE
+
+This enum indicates the sorting type used in the node name list.
+
+```c
+enum NNE_NODENAME_SORTTYPE
```
Comment (Radfordhound) [inline hedgedocs/docs/nn/common/chunk-format/nodenamelist.md:49]: Should probably specify what base type this uses. I forget if this is a uint32_t or a uint8_t that's followed by padding? Only matters for big-endian files.

If you're not sure, don't worry about it, we can just assume it's a uint32_t.

File: hedgedocs/docs/nn/common/chunk-format/nodenamelist.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## NNS_NODENAME
+
+The actual node (bone) name
+
+```c
+struct NNS_NODENAME
+{
+    /* The node index */
+    int iNode;
```
Comment (Radfordhound) [inline hedgedocs/docs/nn/common/chunk-format/nodenamelist.md]: Should probably be int32_t ? (Minor nitpick)

**Review by Radfordhound — CHANGES_REQUESTED** [review]
Looks awesome, thanks a lot for helping out!

There's two very minor changes I'd like to make to make it a little easier for devs reading this to tell what the sizes of the fields are. Then, LGTM

**Comment by TheExentist151** [comment]
Thanks for the review! I changed the type of `iNode` in `NNS_NODENAME` struct to uint32_t, because the `nNode` variable is uint32_t too. 

I can't really check the base type of the `NNE_NODENAME_SORTTYPE` struct, but I think this is just a uint32_t.

**Comment by Radfordhound** [comment]
Thanks, I appreciate it! The base type thing is fine; if we figure it out someday we can just add it in later.

**Review by Radfordhound — APPROVED** [review]

**Pull request merged**